aicsimageio/readers/czi_reader.py

```diff
@@ -4,6 +4,7 @@
 import xml.etree.ElementTree as ET
 from copy import copy
 from typing import Any, Dict, Hashable, List, Optional, Tuple, Union
+import warnings
```

Comment:
Nit: this should be before any `from` imports

Reply:
Yep! Linter got it too!